### PR TITLE
Fix comment on CancellationTokenRegistration.Token

### PR DESCRIPTION
**Pull request by stephentoub**

### CODE REVIEW

**Comment by stephentoub** [comment]
@ViktorHofer, @safern, I assume the answer is "no", but do we have the ability to detect when a change only impacts comments?  It'd be nice to be able to avoid running test legs in such cases and only run a minimal set of build legs.

**Comment by safern** [comment]
> @ViktorHofer, @safern, I assume the answer is "no", but do we have the ability to detect when a change only impacts comments? It'd be nice to be able to avoid running test legs in such cases and only run a minimal set of build legs.

Unfortunately we don't have such mechanism. However, we control which legs are runs via variables and running: https://github.com/dotnet/runtime/blob/master/eng/pipelines/evaluate-changed-paths.sh

We could expand that script to do more validation about the changes, or introduce a feature that if the commit contains a prefix `[Skip Tests]` then it produces a variable that we use to condition test runs. Just throwing some ideas out, if this sounds promising, we could open an issue and discuss it with the infra team.

File: src/libraries/System.Private.CoreLib/src/System/Threading/CancellationTokenRegistration.cs [hunk 1]
```diff
@@ -53,11 +53,12 @@ public ValueTask DisposeAsync()
                 default;
         }
 
-        /// <summary>
-        /// Gets the <see cref="CancellationToken"/> with which this registration is associated.  If the
-        /// registration isn't associated with a token (such as after the registration has been disposed),
+        /// <summary>Gets the <see cref="CancellationToken"/> with which this registration is associated.</summary>
```
Comment (safern) [inline src/libraries/System.Private.CoreLib/src/System/Threading/CancellationTokenRegistration.cs:56]: Should these updates be included in `dotnet-api-docs` repo as well? cc: @carlossanlop

**Review by safern — APPROVED** [review]

File: src/libraries/System.Private.CoreLib/src/System/Threading/CancellationTokenRegistration.cs (hunk 1 above)
Comment (stephentoub) [reply src/libraries/System.Private.CoreLib/src/System/Threading/CancellationTokenRegistration.cs:56]: Yes:
https://github.com/dotnet/dotnet-api-docs/pull/4186
:Smile:

Comment (safern) [reply src/libraries/System.Private.CoreLib/src/System/Threading/CancellationTokenRegistration.cs:56]: Awesome 🚀

**Pull request merged**